modules/round/src/main/Env.scala

```diff
@@ -125,6 +125,12 @@ final class Env(
 
   lazy val proxyRepo: GameProxyRepo = wire[GameProxyRepo]
 
+  scheduler.scheduleAtFixedRate(1 minute, 1 minute) { () =>
+    prefApi.corresEmailNotifUsers flatMap userRepo.byIds flatMap proxyRepo.correspondenceGamesToPlayOf foreach {
+      _ foreach { opps => Bus.publish(opps, "dailyCorrespondenceNotif") }
```

Comment:
publishing an event per user instead of one for all users, not sure if it changes anything